Kettle Judicial Update

ArcanissKenku opened the pull request:
- Added Justice Department mapped to Kettle
- Added vents/scrubbers to Zoo enclosures
- Added APC's for IPC recharging to dorms

# Description

Adds Judicial Department to Kettle

Have also added vents/scrubbers to Zoo enclosures (to prevent animals dying from airloss eventually), IPC recharging APCs in Dorms and minor cosmetic changes.

---
<details><summary><h1>Media</h1></summary>
<p>
New Judicial Department
<img width="1055" height="861" alt="kettleupdate" src="https://github.com/user-attachments/assets/434e48dd-de15-4283-970e-88ea89b7d0ed" />

New Kettle Render
<img width="5696" height="6560" alt="Kettle Render" src="https://github.com/user-attachments/assets/7d536016-f8e6-4d6a-baee-49a7a3060847" />

</p>
</details>

---

# Changelog

:cl:
- add: Judicial Department added to Kettle

ArcanissKenku committed:
Kettle Update

- Added Justice Department mapped to Kettle
- Added vents/scrubbers to Zoo enclosures
- Added APC's for IPC recharging to dorms

Dunrab commented:
@ArcanissKenku The airlock in the court house is AirlockMaintLocked, meaning anyone can open it and just walk into the court room, need to change that to AirlockJusticeLocked since Justice dosent have its own maints airlock.

Also since you touched up chemistry a tiny bit could you also replace the paramed lockers? They are currently LockerParamedicFilled and don't provide any hardsuits for the parameds and need to be swapped to LockerParamedicFilledHardsuit.

ArcanissKenku committed:
Kettle Update

- Fixed Airlock to Courtroom, Justice Locked
- Added Hardsuits to Paramed Lockers
- Added Holopads

ArcanissKenku commented:
Thanks @Dunrab - changes made

Dunrab reviewed (approved):
Now that the airlocks fixed changes are good to go.
